npm-requirements.txt

```diff
@@ -1,7 +1,4 @@
-remark@13.0.0
 remark-cli@9.0.0
-remark-lint@8.0.0
```

Comment:
- remark-cli depends on remark --> so no need to explicitly install remark ... remark-cli suffices. see [package.json](https://github.com/remarkjs/remark/blob/80658e9e601452f0d97fa71ba8b7ba1ed43cdec7/packages/remark-cli/package.json#L33)
- when using "presets", remark-lint does not have to be installed separately, see: [readme](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint) and quote below

> If you’re using presets, they already include remark-lint itself. If you’re using just plugins, you have to include remark-lint explicitly.